lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs

```diff
@@ -193,9 +193,9 @@
                 <p>Dear <%=firstName%> <%=lastName%>,</p>
 
                 <% if (isMultipleSubServicesReq) { %>
-                    <p>New sub-services have been added to your DfE Sign-in account.</p>
+                    <p>Your request to change sub-services access has been approved.</p>
```

Comment:
In the prototype we have only `Your request to change sub-service access has been approved.` - if already checked with the UI/UX designer, please ignore this comment

Reply:
checked with matt when we had the some sub-services only one item and some have multiples and noticed that in certain parts of the html and text it looks for multiples he said if we can then we should

Reply:
this is done